src/file.cpp

```diff
@@ -0,0 +1,26 @@
+#include "rubymotion.h"
+
+/// @class FileUtils
+
+VALUE rb_cFileUtils = Qnil;
+
+/// @method #read
+/// Open a file and return its content
+/// @return [String] the content.
+
+static VALUE
+file_read(VALUE rcv, SEL sel, VALUE val)
+{
+    std::string filename = RSTRING_PTR(val);
+    auto content = cocos2d::FileUtils::getInstance()->getStringFromFile(filename);
+    return RSTRING_NEW(content.c_str());
+}
+
+extern "C"
+void
+Init_FileUtils(void)
+{
+    rb_cFileUtils = rb_define_class_under(rb_mMC, "FileUtils", rb_cObject);
```

Comment:
May I suggest we name this class `File` instead of `FileUtils`? Since `File.read` is a valid in CRuby, we would then have a matching `MG::File.read` method.